tutorials/modelcard-tribuo-v4.ipynb

```diff
@@ -0,0 +1,485 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# ModelCard Tutorial\n",
+    "\n",
+    "Even with the increasing use of ML systems, proper documentation of such systems has yet to gain traction. This has often been linked to the absence of standardized documentation procedures, which has recently led to the proposal of new documentation frameworks for ML models. One example includes the Model Card project, proposed by M. Mitchell et al. in \"Model Cards for Model Reporting\" (2019). This framework aims to allow more transparent model reporting by having the model creator document numerous suggested details (e.g., training algorithms, intended users, etc.). \n",
```

Comment:
I think you should add a link to the model cards paper (e.g. https://dl.acm.org/doi/10.1145/3287560.3287596) which will make it easy to find when this tutorial is published on the tribuo.org website.